scapy/volatile.py

```diff
@@ -930,7 +930,7 @@ def _fix(self):
 # Automatic timestamp
 
 
-class AutoTime(VolatileValue):
+class AutoTime(RandNum):
```

Comment:
`AutoTime` is a number but calling `int()` would make it crash. Tested through the docs